tools/clean.sh

```diff
@@ -34,10 +34,17 @@ rm -f $SRCDIR/config/Kv_Shared*
 cd $SRCDIR/examples/common/python
 make clean
 
+# --------------- EXAMPLES ENCLAVE MANAGER ---------------
+cd $SRCDIR/examples/enclave_manager
+SGX_MODE=${SGX_MODE:-SIM} make clean
+
 # --------------- TOP COMMON ---------------
 #cd $SRCDIR/common
 #make clean
```

Comment:
Can you remove commented code( line #41 - #43)?

Reply:
done as fb6f4222938977f83a5c9dec258ea0d3e2a7a0f6 @manju956